src/Equinor.ProCoSys.IPO.Query/GetInvitationById/InvitationDto.cs

```diff
@@ -39,6 +41,7 @@ public class InvitationDto
         public string CreatedBy { get; }
         public DateTime StartTimeUtc { get; }
         public DateTime EndTimeUtc { get; }
+        public Guid ObjectGuid { get; }
```

Comment:
Do any clients need ObjectGuid?

Reply:
.. I guess you've added ObjectGuid here since you think you need it in UnAcceptPunchOutCommand?
That's incorrect in my head, because you already have the invitationId in UnAcceptPunchOutCommand and can get ObjectGuid for that Invitation from DB. So your invitationValidator.SameUserUnAcceptingThatAcceptedAsync should take invitationId and not ObjectGuid